src/utils/useUpdateEffect/useUpdateEffect.ts

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
If you add `/* eslint-disable prefer-arrow-callback */` here that'll kill the eslint errors about preferring arrow functions

Reply:
I'm very confuse. Where are you seeing these eslint errors?

Reply:
Afaict all the CI has passed

Reply:
Looks like this is an issue with VSC on my machine, so going to resolve this and move along.